brainiak/utils/utils.py

```diff
@@ -964,7 +964,6 @@ def array_correlation(x, y, axis=0):
     r : float or 1D ndarray
         Pearson correlation values for input variables
     """
-
```

Comment:
Please remove this one whitespace difference as well.

Reply:
I think that's done.